Bottom: 484647ef80bfb01c3a7a609d0a7ea5d6d740da67
Top:    2473f82506619b7ecac0d573384d182bb00869bf
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:51:00 -0500

Update configuration for running behind reverse proxy



---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index d23f6eee7..656c05a3e 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -47,7 +47,7 @@ container_env:
   BUILDBOT_SMTP_USER: 'buildbot'
   BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
-  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
+  BUILDBOT_WEB_URL: 'https://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
